src/AbstractMenu.js

```diff
@@ -65,6 +65,11 @@ export default class AbstractMenu extends Component {
         }
     }
 
+    hideMenu = (e) => {
```

Comment:
This is implemented here https://github.com/vkbansal/react-contextmenu/blob/master/src/ContextMenu.js#L156

Reply:
You'll need to handle this in `SubMenu` and not here.

Reply:
@vkbansal Updated it in subMenu